mail/migrations/0002_default_email_templates.py

```diff
@@ -0,0 +1,104 @@
+from django.db import migrations, models
+
+
+def forwards_func(apps, schema_editor):
+    """
+    It adds default email templates
+    """
+    AutomaticReminderEmail = apps.get_model("mail", "AutomaticReminderEmail")
+    db_alias = schema_editor.connection.alias
+
+    # first email for reminders to pay one week before deadline
+    AutomaticReminderEmail.objects.using(db_alias).create(
+        days_before=7,
+        email_subject="Payment due for %recipient.bootcamp_name% Bootcamp",
+        email_body="""Dear %recipient.full_name%,
+
+Your tuition payment for %recipient.bootcamp_name% is due. \
```

Comment:
Add `Bootcamp` here similar to the subject line, i.e. `... for %recipient.bootcamp_name% Bootcamp is due`